scripts/archive_api.py

```diff
@@ -77,6 +107,57 @@ async def health_check():
 	        "HopAuthOK": hop_auth_ok,
 	       }
 
+messageRecordSchema = {
+                          "type": "object",
+                          "properties": {
+                              "message": {
+                                  "type": "string",
+                                  "format": "binary",
+                                  "description": "The raw mesage body"
+                              },
+                              "metadata": {
+                                  "type": "object",
+                                  "description": "Kafka metadata for the message",
+                                  "properties": {
+                                      "timestamp": {
+                                          "type": "number",
+                                          "description": "The time at which the message was sent to the Kafka broker"
+                                      },
+                                      "headers": {
+                                          "type": "object",
+                                          "description": "Kafka message headers",
+                                          "patternProperties": {
```

Comment:
This is a little nitpicky but it might be nice to call out some of the more used headers explicitly in the docs here. In hermes we use `_sender` and `_id` at least